tests/fingerprints_tests.py

```diff
@@ -18,21 +56,165 @@ def test_morgan_fingerprint():
         f"./test_datasets/{'_'.join(dataset_name.split('-'))}/mapping/mol.csv.gz"
     )
     X = dataset["smiles"]
+    X = X[:100]
 
     # Copy X for second sequential calculation
     X_2 = X.copy()
 
     # Concurrent
-    morgan = MorganFingerprintAsBitVect(radius=2, n_bits=2048, n_jobs=-1)
-    X_train = morgan.transform(X)
+    morgan = MorganFingerprint(result_type="default", n_jobs=-1)
+    morgan_hashed = MorganFingerprint(result_type="hashed", n_jobs=-1)
```

Comment:
To powinny być 2 osobne testy, bo to 2 różne fingerprinty (technicznie)

Reply:
Czy chodzi tutaj, że każdy fingerprint, który ma wersję hashed mam rozdzielić, czy tylko w przypadku Morgana?